src/lights/PointLight.js

```diff
@@ -10,6 +10,7 @@ THREE.PointLight = function ( color, intensity, distance ) {
 
 	this.intensity = ( intensity !== undefined ) ? intensity : 1;
 	this.distance = ( distance !== undefined ) ? distance : 0;
+	this.decayExponent = ( decayExponent !== undefined ) ? decayExponent : 1;	// for physically correct lights, should be 2.
```

Comment:
Even if we would "break" stuff. Should we use 2 as default so things look better?
Also, would call this variable just `decay` be too confusing?